frameworks/react/src/AdvancedVideo.tsx

```diff
@@ -32,6 +33,10 @@ interface VideoProps {
   onEnded?: ReactEventHandler<any>
 }
 
+interface VideoProps extends BaseVideoProps {
+  innerRef?: ((instance: unknown) => void) | MutableRefObject<unknown> | null
```

Comment:
Please use `any` instead of unknown, since `unknown` is oddly enough unknown to typescript 2.x :) (Not supported, it was introduced in 3.x)